linode/manager
# upcoming: [M3-7726] - Assign Linodes to Placement Group Drawer

## Description 📝
This PR brings the drawer to assign linodes to a PG (in the VPC subnet fashion) + improvements and naming convention updates

## Changes  🔄
- Add new drawer and tests
- Connect drawer to flow
- Update utils and function names to use "linode" instead of "vm"
- Small (non breaking) updates to the RemovableSelectionList to allow for extra styling and accessibility improvements 

## Preview 📷
![Screenshot 2024-02-07 at 13 29 51](https://github.com/linode/manager/assets/130582365/c801ee2e-7629-4a3b-a467-0a514b1f1158)


## How to test 🧪

### Prerequisites
- Turn on both the "Placement Groups" feature flag and MSW

### Verification steps 
- Navigate to /placement-groups/1/linodes
- Click on "Add Linode to Placement Group"
- Confirm 
  - overall form behavior
  - adding/removing a linode from the list (and invalidation)

**NOTE**:  due to limitations in using MSW, we can only really see adding/removing ONE linode at a time (subsequent tries won't be reflected in the UI)

## As an Author I have considered 🤔

*Check all that apply*
